sdk/core/azure-core/src/main/java/com/azure/core/credential/SimpleTokenCache.java

```diff
@@ -64,8 +64,8 @@
  */
 public class SimpleTokenCache {
     // The delay after a refresh to attempt another token refresh
-    private static final Duration REFRESH_DELAY = Duration.ofSeconds(30);
-    private static final String REFRESH_DELAY_STRING = String.valueOf(REFRESH_DELAY.getSeconds());
+    private static Duration refreshDelay = Duration.ofSeconds(30);
```

Comment:
This is not updated outside the test code. Do we want this to be mutable? Can we make this static final? If we want this to be mutable, we should make this an instance variable instead of static var.